Microsoft.Azure.CosmosRepository/src/DefaultRepository.cs

```diff
@@ -279,6 +279,39 @@ public async ValueTask<bool> ExistsAsync(Expression<Func<TItem, bool>> predicate
             return count > 0;
         }
 
+        public async ValueTask<IPage<TItem>> ScrollAsync(int lastPage = 0,
+            Expression<Func<TItem, bool>> predicate = null,
+            int pageSize = 25,
+            string continuationToken = null,
+            CancellationToken cancellationToken = default)
+        {
+            Container container = await _containerProvider.GetContainerAsync().ConfigureAwait(false);
+
+            QueryRequestOptions options = new()
+            {
+                MaxItemCount = pageSize
+            };
+
+            IQueryable<TItem> query = container.GetItemLinqQueryable<TItem>(requestOptions: options, continuationToken: continuationToken)
+                .Where(_repositoryExpressionProvider.Build(predicate));
+
+            Response<int> countResponse = await query.CountAsync(cancellationToken);
+
+            List<TItem> results = new();
+
+            FeedIterator<TItem> iterator = query.ToFeedIterator();

```

Comment:
Are you missing a `using` here, I'm not sure - you'd have to verify (also check if it is an imple?

```suggestion
            using FeedIterator<TItem> iterator = query.ToFeedIterator();
```